Apps/Sandcastle/gallery/3D Tiles Batch Table Hierarchy.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
```

Comment:
Depending on what time you run all the 3D Tiles Sandcastle examples, the shading might not be great:

![image](https://cloud.githubusercontent.com/assets/782098/26328327/7bd04464-3f10-11e7-87c7-95eb488b83cb.png)

Should the examples explicitly set a reasonable time?